tests/func/test_remove.py

```diff
@@ -32,10 +32,40 @@ def test_remove(tmp_dir, scm, dvc, run_copy, remove_outs):
     assert not (tmp_dir / ".gitignore").exists()
 
 
+def test_remove_file_as_target(tmp_dir, scm, dvc):
+    dvc.run(
+        name="my",
+        cmd='echo "hello" > foo && echo "hello" > bar',
+        deps=[],
+        outs=["foo", "bar"],
+    )
+
+    dvc.remove("foo")
+    assert "/foo" not in get_gitignore_content()
+    assert "/bar" in get_gitignore_content()
+
+    dvc.remove("bar")
+    assert not (tmp_dir / ".gitignore").exists()
+
+
+def test_remove_file_in_subdir_as_target(tmp_dir, scm, dvc):
```

Comment:
These two tests can be merged into one? Generating a stage output both file and path, then remove them.

Reply:
I felt like the 2 tests are better separated because one is asserting a successful behavior and the other is asserting that a specific exception is being raised